index.html

```diff
@@ -0,0 +1,193 @@
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+      </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+      </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+        </div>
+        </li>
+      </ul>
+    </section>
+
+    <section id="about-me">
+      <article id="about-article">
+        <h1>About me</h1>
+        <p id="about-text"> I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button type="submit">Get My Resume</button>
+      </article>
+      <ul class="about-row">
+        <li class="about-item">
+          <img class="about-img" src="./pics/icon-languages.svg" alt="icon-languages">
+          <h3>Languages</h3>
```

Comment:
Kindly fix the font size of these headings for all `about-items` according to Figma Design 🤗
  -  _Make the font-size 2rem._

Your design:

![image](https://user-images.githubusercontent.com/73714615/187035994-c87d424b-d67a-437e-8575-3a7aab1d47cb.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187035973-095493ef-d58a-406a-a45b-ffaff722f7fa.png)